### DISCUSSION TITLE
need to add jquery validation

### DISCUSSION THREAD

**Discussion by Theni-N-Lingeswaran** [sa-si-dev/virtual-select#68] [Q&A] [answered]
how can i add validation for virtual select plugin..?

**Comment by sa-si-dev** [comment]
> how can i add validation for virtual select plugin..?

I will try and let you know.

**Comment by sa-si-dev** [comment, marked as answer]
@Theni-N-Lingeswaran it seems jquery validation plugin doesn't support custom elements like `div` which is used by Virtual Select plugin.
So we can't use jquery plugin with Virtual Select for now.

But Virtual Select has `required` validtion. You can use it along with other validation plugin.

For more details refer https://sa-si-dev.github.io/virtual-select/#/examples?id=validation
